pipeline/outputs/elasticsearch.md

```diff
@@ -8,54 +8,59 @@ The **es** output plugin, allows to ingest your records into an [Elasticsearch](
+| AWS\_STS\_Endpoint | Specify the custom sts endpoint to be used with STS API for Amazon OpenSearch Service |  | Yes |
+| AWS\_Role\_ARN | AWS IAM Role to assume to put records to your Amazon cluster |  | Yes |
+| AWS\_External\_ID | External ID for the AWS IAM Role specified with `aws_role_arn` |  | Yes |
+| AWS\_Service\_Name | Service name to be used in AWS Sigv4 signature. For integration with Amazon OpenSearch Serverless, set to `aoss`. See the [FAQ](opensearch.md#faq) section on Amazon OpenSearch Serverless for more information. | es | Yes |
+| AWS\_Profile | AWS profile name | default | Yes |
+| Cloud\_ID | If you are using Elastic's Elasticsearch Service you can specify the cloud\_id of the cluster running. The Cloud ID string has the format `<deployment_name>:<base64_info>`. Once decoded, the `base64_info` string has the format `<deployment_region>$<elasticsearch_hostname>$<kibana_hostname>`. |  | No |
+| Cloud\_Auth | Specify the credentials to use to connect to Elastic's Elasticsearch Service running on Elastic Cloud |  | Yes |
+| HTTP\_User | Optional username credential for Elastic X-Pack access |  | Yes |
+| HTTP\_Passwd | Password for user defined in HTTP\_User |  | Yes |
+| Index | Index name | fluent-bit | Yes |
+| Type | Type name | \_doc | Yes |
+| Logstash\_Format | Enable Logstash format compatibility. This option takes a boolean value: True/False, On/Off | Off | Yes |
+| Logstash\_Prefix | When Logstash\_Format is enabled, the Index name is composed using a prefix and the date, e.g: If Logstash\_Prefix is equals to 'mydata' your index will become 'mydata-YYYY.MM.DD'. The last string appended belongs to the date when the data is being generated. | logstash | Yes |
+| Logstash\_Prefix\_Key | When included: the value of the key in the record will be evaluated as key reference and overrides Logstash\_Prefix for index generation. If the key/value is not found in the record then the Logstash\_Prefix option will act as a fallback. The parameter is expected to be a [record accessor](../../administration/configuring-fluent-bit/classic-mode/record-accessor.md). |  | Yes |
+| Logstash\_Prefix\_Separator | Set a separator between logstash_prefix and date.| - | Yes |
+| Logstash\_DateFormat | Time format \(based on [strftime](http://man7.org/linux/man-pages/man3/strftime.3.html)\) to generate the second part of the Index name. | %Y.%m.%d | Yes |
+| Time\_Key | When Logstash\_Format is enabled, each record will get a new timestamp field. The Time\_Key property defines the name of that field. | @timestamp | Yes |
+| Time\_Key\_Format | When Logstash\_Format is enabled, this property defines the format of the timestamp. | %Y-%m-%dT%H:%M:%S | Yes |
+| Time\_Key\_Nanos | When Logstash\_Format is enabled, enabling this property sends nanosecond precision timestamps. | Off | Yes |
+| Include\_Tag\_Key | When enabled, it append the Tag name to the record. | Off | Yes |
```

Comment:
```suggestion
| Type | Type name | `\_doc` | Yes |
| Logstash\_Format | Enable Logstash format compatibility. This option takes a boolean value: True/False, On/Off. | `Off` | Yes |
| Logstash\_Prefix | When Logstash\_Format is enabled, the Index name is composed using a prefix and the date. For example, if Logstash\_Prefix is `mydata`, your index becomes `mydata-YYYY.MM.DD`. The last string appended belongs to the date when the data is being generated. | `logstash` | Yes |
| Logstash\_Prefix\_Key | When included, the value of the key in the record will be evaluated as key reference and overrides Logstash\_Prefix for index generation. If the key/value isn't found in the record, the Logstash\_Prefix option will act as a fallback. The parameter is expected to be a [record accessor](../../administration/configuring-fluent-bit/classic-mode/record-accessor.md). | _none_ | Yes |
| Logstash\_Prefix\_Separator | Set a separator between logstash_prefix and date. | `-` | Yes |
| Logstash\_DateFormat | Time format \(based on [strftime](http://man7.org/linux/man-pages/man3/strftime.3.html)\) to generate the second part of the Index name. | `%Y.%m.%d` | Yes |
| Time\_Key | When Logstash\_Format is enabled, each record gets a new timestamp field. The Time\_Key property defines the name of that field. | `@timestamp` | Yes |
| Time\_Key\_Format | When Logstash\_Format is enabled, this property defines the format of the timestamp. | `%Y-%m-%dT%H:%M:%S` | Yes |
| Time\_Key\_Nanos | When Logstash\_Format is enabled, enabling this property sends nanosecond precision timestamps. | `Off` | Yes |
| Include\_Tag\_Key | When enabled, it append the Tag name to the record. | `Off` | Yes |
```

Reply:
Suggestion with small fix (there is no need to escape underscore with backslash if underscore is wrapped with backticks) and modification (`Off` and `On` can be considered special values and not literals, so `Off` and `On` are not wrapped with backticks) was applied in c620db5a2ca5a0cd0f5a6474e0db8a0139bca3c3, cdde1dec1e45673d9a9554429feea6df14f113c0 and 929d05ddfc37113904775d9e80e0d6f1fefcb1a9.